### ISSUE TITLE
LocalizedControlTypeReasonable fails with German locale

### ISSUE THREAD

**Issue by ollypsilon** [microsoft/accessibility-insights-windows#1572]
The following accessibility issue needs investigation.

**App:** MSACCESS

**Element path:** Bild 'Mit Access verknüpfte Tabelle'

**Issue Details:** The LocalizedControlType should be reasonable based on the element's ControlTypeId. [Section 508 502.3.1](https://www.access-board.gov/ict/#502-interoperability-assistive-technology)

**How To Fix:** Set a meaningful LocalizedControlType UI Automation property for this element.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

Beside automatic issue text above here's the element's details:
![image](https://user-images.githubusercontent.com/52780811/224994777-d9d54a5c-63ec-417c-89b3-abe72836f3ea.png)

**Comment by codeofdusk** [comment]
“Bild” is [image in German](https://en.wiktionary.org/wiki/Bild#German), so this appears to be a false positive .

**Comment by DaveTryon** [comment]
Some more info here: Neither Axe.Windows nor AIWin currently support running with non-English localized resources, so in this case the rule is comparing the German-localized name to the English-localized name and not finding a match.

We could potentially alter the rule to exclude scenarios where the locale language doesn't match the process language, but we'd need to work out the details of how this works, how to document it, etc.

**Comment by DaveTryon** [comment]
We've identified 2 options here: Either:
1. Disable this rule if the language is not English OR
2. Provide a localized dictionary of appropriate names and check the localized name. If we don't have a localized version available, then skip the rule

**Comment by DaveTryon** [comment]
Plan is to disable this rule if running on a non-English language setting

**Comment by DaveTryon** [comment]
This has been address in Axe.Windows 2.1.2.

**Comment by DaveTryon** [comment]
@ollypsilon, this is fixed in our Canary release available at at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.2334.01. It will soon ship to Production.

**Comment by DaveTryon** [comment]
Per our policy, closing the issue as it's available in Canary.
